core/src/test/java/org/apache/calcite/test/ReflectiveSchemaTest.java

```diff
@@ -1070,4 +1070,13 @@ public IntHolder(int value) {
             + "from \"s\".\"primesCustomBoxed\"")
         .returnsUnordered("EXPR$0=false\nEXPR$0=false\nEXPR$0=true");
   }
+
+  @Test public void testUnaryMinus() {
```

Comment:
testDecimalNegate?
Please also check the codegen with `planContains("....negeate()"`

Reply:
OK